website/source/docs/drivers/external/pot-task-driver.html.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: "docs"
+page_title: "Drivers: pot-task-driver"
+sidebar_current: "docs-drivers-community-pot-task-driver"
+description: |-
+  The Pot task driver is used to run pot (https://github.com/pizzamig/pot) containers using FreeBSD jails.
+---
+
+# Pot task Driver
+
+Name: `pot-task-driver`
```

Comment:
The name we use here, as the documentation page title, and table of contents, should match the name that the driver registers (`pot`, in this case). I think it's only `jail-task-driver` that's the oddball here using "task-driver" in the name.